src/globus_cli/commands/endpoint/permission/update.py

```diff
@@ -25,17 +29,18 @@
 @click.argument("rule_id")
 @click.option(
     "--permissions",
-    required=True,
     type=click.Choice(("r", "rw"), case_sensitive=False),
     help="Permissions to add. Read-Only or Read/Write",
 )
+@expiration_date_option
 @LoginManager.requires_login("transfer")
 def update_command(
     login_manager: LoginManager,
     *,
-    permissions: t.Literal["r", "rw"],
+    permissions: t.Literal["r", "rw"] | None,
     rule_id: str,
     endpoint_id: uuid.UUID,
+    expiration_date: str | None,
```

Comment:
Is it worth ensuring that at least _one_ of {`permissions` | `expiration_date`} is set? Not sure what the behavior of the API is in this case; if it results in a nice enough error via the CLI, then maybe we leave as-is, but wanted to check since we're loosening the input expectations.

Reply:
This is the response with neither option given. It seems clear enough to me, but also wouldn't be too hard to require one of the options if that would be better.

```
Globus CLI Error: A Transfer API Error Occurred.
HTTP status:      400
request_id:       ugM0HuK7L
code:             BadRequest
message:          No fields were specified to update
```

Reply:
I'm waffling (my preference to delegate as much as possible to the service is competing with my desire to give the most helpful error message to the user), but I think I'm okay with it as-is, knowing that we could easily revisit the topic later if we've got it wrong!